packages/core/src/modules/dids/__tests__/__fixtures__/didKeyBls12381g1.json

```diff
@@ -1,5 +1,5 @@
 {
-  "@context": ["https://w3id.org/did/v1"],
+  "@context": ["https://w3id.org/did/v1", "https://w3id.org/security#Bls12381G1Key2020"],
```

Comment:
This is not a valid context url. If this is the key specific url we should use the suite url instead.

Reply:
Fixed in a18ab0d3.